cobalt/loader/image/animated_image_tracker.cc

```diff
@@ -140,13 +141,27 @@ void AnimatedImageTracker::Reset() {
 }
 
 void AnimatedImageTracker::OnRenderTreeRasterized() {
+  bool is_main_webmodule = name_ == "MainWebModule";
   count_animated_images_active = playing_urls_.size();
+  if (is_main_webmodule) {
+    UmaHistogramExactLinear("AnimatedImage.MainWebModule.Active",
+                            count_animated_images_active, UINT_MAX);
+  }
   for (const auto& playing_url : playing_urls_) {
     auto image = image_map_[playing_url.first].get();
     auto stats = image->GetFrameDeltaStats();
     count_animated_frames_decoded += stats.frames_decoded;
     count_animated_frames_decoding_underrun += stats.frames_underrun;
     count_animated_frames_decoding_overrun += stats.frames_overrun;
+    if (is_main_webmodule) {
+      UmaHistogramExactLinear("AnimatedImage.MainWebModule.DecodedFrames",
```

Comment:
This isn't right. I should probably be tracking a rate, not an absolute value here and below.

Reply:
I think i worked out most of what we want to know in the JavaScript demo above in the previous PR. I'll land that first and confirm with YTS folks what we have there is sufficient. Then adjusting these here should become obvious.

Again, this was a 5-minute commit draft to give you a heads up